k8s/secret.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: django-secrets
+  namespace: forum
+type: Opaque
+stringData:
+  # Django
+  SECRET_KEY: "django-insecure-7z@6p(u8+pz%q=nb8-xf^b4#e*4%-+mgo0=3_3wlvhh3hf4r+p"
+
+  # Database
+  DB_PASSWORD: "postgres"
+
+  # Email
+  EMAIL_HOST_USER: ""
```

Comment:
EMAIL_HOST_USER is empty, which may cause runtime errors. Provide a non-empty default or set via environment variable.
```
EMAIL_HOST_USER: "<your-email@example.com>"
```